code/integration-tests/runtime-tests/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "runtime-tests",
+  "version": "1.0.0",
+  "description": "devnet tests",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "tsc; ts-mocha --parallel ./tests/*Tests.ts"
```

Comment:
so it runs parallel and is fine? awesome than.

Reply:
Currently there is only one test but each test is using different wallets. The only problem might occur from sudo but I think I can move presteps into a generic before file to be run before.